src/descriptors/power_system_structs.json

```diff
@@ -1983,12 +1988,14 @@
             "min": 0,
             "max": null
           },
-          "validation_action": "error"
+          "validation_action": "error",
+          "default": "nothing"
         },
         {
           "name": "operation_cost",
           "null_value": "ThreePartCost(nothing)",
-          "data_type": "ThreePartCost"
+          "data_type": "ThreePartCost",
+          "default": "ThreePartCost(nothing)"
```

Comment:
does this introduce functionality that relies on the "non-functional" demo constructor
https://github.com/NREL-SIIP/PowerSystems.jl/blob/228ef8dd5765640dee30f7c4e9da924223ea30ab/src/models/generated/ThreePartCost.jl#L47

Would it be preferable to add defaults to `ThreePartCost`?

More generally, i wonder if all `Foo(::Nothing)` methods could be replaced by `Foo()` and appropraite defaults (which would them also be useful to the keyword constructors that override some but not all defaults)

Reply:
The same issue is here, there are reasonable estimates for this. Even if it is zero. Having a `nothing` implies that the information doesn't exist or there is no reasonable estimates.

Reply:
I have removed the default for this field as requested by @jd-lara